setup.py

```diff
@@ -6,7 +6,7 @@
 
 
 def load_readme():
-    with io.open(os.path.join(HERE, "README.rst"), "rt", encoding="utf8") as f:
+    with io.open(os.path.join(HERE, "README.md"), "rt", encoding="utf8") as f:
```

Comment:
Changing the README format to Markdown requires that you add `long_description_content_type='text/markdown'` to the `setup()` method's parameters.

Reply:
Good catch :+1: will do